ani-cli

```diff
@@ -438,6 +439,11 @@ while [ $# -gt 0 ]; do
 done
 [ "$use_external_menu" = "0" ] && multi_selection_flag="${ANI_CLI_MULTI_SELECTION:-"-m"}"
 [ "$use_external_menu" = "1" ] && multi_selection_flag="${ANI_CLI_MULTI_SELECTION:-"-multi-select"}"
+if [ "$external_menu_normal_window" = "1" ]; then
+    external_menu_args="-normal-window"
+else
+    external_menu_args=""
+fi
```

Comment:
The default value for an unbound variable in shell is already the empty string and references do not result in an error unless `set -u` is set. [(source)](https://unix.stackexchange.com/questions/463034/bash-throws-error-line-8-1-unbound-variable)
```suggestion
[ "$external_menu_normal_window" = "1" ] && external_menu_args="-normal-window"
```